doc/rtd/reference/ubuntu_stable_release_updates.rst

```diff
@@ -0,0 +1,49 @@
+.. _stable_release_updates:
```

Comment:
I liked that we provided specific instructions on how to install proposed and then run a test on that proposed instance. At the very least I think we should link to a procedure to install a package from proposed (I'm assuming one exists somewhere on the internet?)

Reply:
I added this information as a how to guide and a link on the page. Let me know if that works for you.

Reply:
Yes, the new version is great!